src/main/kotlin/org/opensearch/observability/index/ObservabilityIndex.kt

```diff
@@ -82,6 +83,14 @@ internal object ObservabilityIndex {
         this.mappingsUpdated = false
     }
 
+    /**
+     * once lifecycle indicate start has occurred - instantiating system index creation
+     */
+    override fun afterStart() {
```

Comment:
When does this get triggered?
Is this after node startup or cluster startup. 
If it is after node startup, is to ok to create an index if other nodes are still booting up.

Reply:
This is triggered from the observability plugin class that implements the cluster listener and registers the started method